scss/_patterns_navigation.scss

```diff
@@ -100,31 +127,37 @@
       }
 
       .p-navigation__link {
```

Comment:
Link external works although blue which is another issue:
![navigation - dark - vanilla framework - examples](https://cloud.githubusercontent.com/assets/1413534/25662568/e3533564-300c-11e7-9c22-204326079b53.png)